agenta-backend/agenta_backend/services/organization_service.py

```diff
@@ -33,3 +55,67 @@ class NotFound(Exception):
     """Custom exception for credentials not found"""
 
     pass
+
+
+def send_invitation_email(
+    email: str, token: str, organization: Organization, user: User
+):
+    message = Mail(
+        from_email="account@hello.agenta.ai",
+        to_emails=email,
```

Comment:
`to_emails` is either a list or tuple, no? 

Did the email sent successfully when you passed a string and not either of this:  `(email,)` or `[email]` ?

Reply:
I think it accepts both. And yes it sent successfully.

Reply:
Cool, cool.